NIX.md

```diff
@@ -0,0 +1,92 @@
+# How to use Nix
+
+## Nix installation
+
+- Install Nix
+  [(learn more about this installer)](https://zero-to-nix.com/start/install)
+
+  ```bash
+  curl --proto '=https' --tlsv1.2 -sSf \
+     -L https://install.determinate.systems/nix \
+     | sh -s -- install
+  ```
+
+## Create GRASS development environment
+
+Nix provides a development environment containing all required dependencies.
+
+- Launch development environment
+
+  ```bash
+  nix develop
+  ```
+
+- Optionally, use [direnv](https://direnv.net) to activate environment
+  automatically when entering the source code directory
+
+  ```bash
+  echo "use flake" > .envrc
+  direnv allow
+  ```
+
+## Launch GRASS directly from the source code
+
+Nix allows to run a program directly from git source code repository using
+following command:
+
+```bash
+nix run \
+  github:<OWNER>/<REPO>/<REVISION|BRANCH|TAG>#<PACKAGE-NAME> -- <PROGRAM-ARGUMENTS>
+```
+
+- Launch latest version of GRASS from `main` branch
+
+  ```bash
+  nix run github:OSGeo/grass#grass
+  ```
+
+- Launch GRASS from specific Git revision, branch or tag
+
+  ```bash
+  nix run github:OSGeo/grass/<REVISION|BRANCH|TAG>#grass
+  ```
```

Comment:
Could you, please, also add an example how to run a GRASS PR?

Reply:
There is no more magic, just run `nix run github:<YOUR-GITHUB-NAME>/<GRASS>/<PR-BRANCH>#grass`. Maybe it makes a sense to document it explicitly.

Reply:
Done.